elements/pfe-navigation-account/src/pfe-navigation-account.js

```diff
@@ -17,6 +17,10 @@ class PfeNavigationAccount extends PFElement {
     };
   }
 
+  static get observedAttributes() {
+    return ["lang"];
```

Comment:
We no longer need to set observedAttributes on the component.  By adding this attribute to the properties object, you can set the default and point it to the observer you want to use.

Reply:
I'd like to decouple needing to upgrade all of PFE & using the new nav. It sounds like this change might be hard to implement in a way that we could support pre-release.

Reply:
If you're planning to publish this component as a 1.x version, I'd disagree.  It really should use the 1.0 PFElement.  We can discuss as a group if you like.